docs/configs/expanse.yaml

```
@@ -0,0 +1,31 @@
+display_name: SDSC Expanse
+engine:
+    max_workers_per_node: 2
```

Although not required, I suggest we match our other config examples and explicitly define the engine type.
```suggestion
engine:
    type: HighThroughputEngine
    max_workers_per_node: 2
```